### ISSUE TITLE
Word "pala" is getting split by nlp

### ISSUE THREAD

**Issue by mllamazares** [explosion/spaCy#5255]
The word "pala" is getting split in two "pa" and "la".

```
import es_core_news_sm
nlp = es_core_news_sm.load()

doc = nlp("pala")
print([(w.text, w.pos_) for w in doc])
```
Result:
`[('pa', 'ADP'), ('la', 'DET')]`

Expected output:
`[('pala', 'NOUN')]`

## Info about spaCy

* **spaCy version:** 2.2.4
* **Platform:** Linux-4.15.0-29deepin-generic-x86_64-with-Deepin-15.11-stable
* **Python version:** 3.6.5

**Comment by svlandeg** [comment]
Hi @mllamazares. spaCy wouldn't normally split a word into pieces like that, but in this case there is an explicit [tokenization exception](https://github.com/explosion/spaCy/blob/master/spacy/lang/es/tokenizer_exceptions.py#L9) that triggers this: 
`"pala": [{ORTH: "pa", LEMMA: "para"}, {ORTH: "la", LEMMA: "la", NORM: "la"}],`

I don't speak Spanish - does this exception make sense in some contexts ?

If it does, we'll want to keep it in the library. But you can overwrite these exceptions in your custom use-case, by e.g removing this one specific rule:

```
custom_exc = nlp.tokenizer.rules
del custom_exc["pala"]
nlp.tokenizer.rules = custom_exc
```

You can find more background in the documentation about [tokenization](https://spacy.io/usage/linguistic-features#tokenization), including more information about these different types of rules, how they are used in spaCy, how you can customize them and how you can create your custom Tokenizer if you'd need it.

One word of warning though; the model you're loading, `es_core_news_sm`, has in fact been trained with the "default" tokenizer already. So it never really saw "pala" as one word during training. Never the less, when I add the above lines to your code snippet, I get:

> [('pala', 'NOUN')]

So I hope this works for you!

**Comment by mllamazares** [comment]
> Hi @mllamazares. spaCy wouldn't normally split a word into pieces like that, but in this case there is an explicit [tokenization exception](https://github.com/explosion/spaCy/blob/master/spacy/lang/es/tokenizer_exceptions.py#L9) that triggers this:
> `"pala": [{ORTH: "pa", LEMMA: "para"}, {ORTH: "la", LEMMA: "la", NORM: "la"}],`
> 
> I don't speak Spanish - does this exception make sense in some contexts ?
> 
> If it does, we'll want to keep it in the library. But you can overwrite these exceptions in your custom use-case, by e.g removing this one specific rule:
> 
> ```
> custom_exc = nlp.tokenizer.rules
> del custom_exc["pala"]
> nlp.tokenizer.rules = custom_exc
> ```
> 
> You can find more background in the documentation about [tokenization](https://spacy.io/usage/linguistic-features#tokenization), including more information about these different types of rules, how they are used in spaCy, how you can customize them and how you can create your custom Tokenizer if you'd need it.
> 
> One word of warning though; the model you're loading, `es_core_news_sm`, has in fact been trained with the "default" tokenizer already. So it never really saw "pala" as one word during training. Never the less, when I add the above lines to your code snippet, I get:
> 
> > [('pala', 'NOUN')]
> 
> So I hope this works for you!

"pa" instead of "para" is correct (Spanish slang), but anyway, it should be separated in a different word by a space, for instance "pa la gente" (for the people). Thus, I think you should fix this structure. According to my understanding, all words should be separated.

**Comment by svlandeg** [comment]
Ok, gotcha. Submitted the fix here: https://github.com/explosion/spaCy/pull/5265 - will be fixed from spaCy 3.0 onwards.
